partcad/src/partcad/ai_openai.py

```
@@ -85,38 +86,43 @@ def generate_openai(
 
         pc_logging.debug("Prompt: %s", prompt)
 
-        image_content = []
-
-        def insert_image(match):
-            filename = match.group(1)
-            image_content.append(
-                {
+        def handle_content(content):
```

Ruff (0.8.2)

Missing return type annotation for private function `handle_content`

(ANN202)